### PR TITLE
JSX for compatibility with latest version of Übersicht

### PR DESCRIPTION
**Pull request by benpetty**
Widget was broken for me after updating

### CODE REVIEW

**Comment by juni-vogt** [comment]
Hi, sorry for replying so late! Would you like to have repository access to this widget and wttr-uebersicht? I don't have a mac to test on anymore ^^'

**Comment by juni-vogt** [comment]
And thanks a lot for submitting your pull request!

File: wttr-moon.widget/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	curl -s wttr.in/moon?lang=en | ./terminal-to-html
+`;
+
+// see https://github.com/chubin/wttr.in for API
+// uses https://github.com/buildkite/terminal
+
+export const render = props => props.error ? props.error :
+	<div>
+		<link rel="stylesheet" href="wttr-moon.widget/terminal-colors.css" />
+		<pre dangerouslySetInnerHTML={{ __html: props.output }} />
```
Comment (juni-vogt) [inline wttr-moon.widget/index.jsx]: I think the `out.err || out.data.split('\n').slice(0, 24).join('\n')` was kind of important for filtering news messages by wttr.in

Comment (benpetty) [reply wttr-moon.widget/index.jsx]: Oh true, much cleaner. Fixed in d406dbabc4b9c1d51ca1fd419c5971b6f8ed5eb8

**Comment by benpetty** [comment]
> Hi, sorry for replying so late! Would you like to have repository access to this widget and wttr-uebersicht? I don't have a mac to test on anymore ^^'

Sure, thanks! Will check the other repo out later 👍

**Comment by juni-vogt** [comment]
Cool :) I sent you invites to both repos so you can change the code freely. https://github.com/matthias-vogt/wttr-moon-uebersicht/invitations BTW, when changing anything you have to repack the zip so that the changes are actually visible.

**Pull request merged**